packs/spell-effects/spell-effect-earthquake-shaking-ground.json

```diff
@@ -0,0 +1,47 @@
+{
+    "_id": "09rtx50laDw68FGT",
+    "img": "systems/pf2e/icons/spells/earthquake.webp",
+    "name": "Spell Effect: Earthquake (Shaking Ground)",
+    "system": {
+        "description": {
+            "value": "<p><em>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Earthquake]</em></p>\n<p>Creatures on the shaking ground take a -2 circumstance penalty to attack rolls, AC, and skill checks.</p>"
```

Comment:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Earthquake]</p>\n<p>Creatures on the shaking ground take a -2 circumstance penalty to attack rolls, AC, and skill checks.</p>"
```

Reply:
Done, for some reason I thought italics was standard there.

Reply:
> Done, for some reason I thought italics was standard there.

It used to be on the spell link, but it’s now handled by the system.